### ISSUE TITLE
Refactor install_gems

### ISSUE THREAD

**Issue by vholer** [OpenNebula/one#4233]
**Description**

Install_gems has (or recently had) few problems and drawbacks
- /usr/share/one/install_gems doesn't install all requirements #3864 
- Don't rely on `lsb_release` for `/usr/share/one/install_gems` #3866
- create cli flags for install_gems #1162 
- ERROR: executing gem invalid option: --no-ri #4211 
- install_gems for 5.10.1 does not include all required gems #4092 

Problem of the script is that it's old, too complex, not very well maintained and with features which are not well known (it even doesn't show help on `--help`). Mainly, it allows installing gems in 2 ways (via `bundler` and directly via `gem`), which results in 3 possible ways how to install OpenNebula dependencies in real deployments: 
1. preffered: our packaged Ruby gems (rpm/deb)
2. install_gems - Bundler (by Gemfile)
3. install_gems - gem (by different list of gems inside the script)

That's crazy. Idea would be to update `install_gems` to
- drop option 3
- depend only on Bundler and manage Gemfile(.locks) as the only source of truth (option 2)
- drop script in current state and reimplement as simple shell script which
    - yum / apt-get installs the dependencies
    - installs bundler
    - triggers bundler

Basically new `install_gems` should look like script https://github.com/OpenNebula/packages/blob/master/rubygems/prepare.sh with extra trigger of `bundler install` at the end. If necessary, gems in Gemfile could be split into groups to allow installation of just part of the dependencies.

Old script could still come with OpenNebula as `install_gems-obsolete` for some time.

**Use case**

Make the process clear and easier to maintain and extend for other platforms.

## Progress Status
- [ ] Branch created
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by al3xhh** [comment]
PRs to merge in 5.12 and master:

- [code](https://github.com/OpenNebula/one-ee/pull/57)
- [docs](https://github.com/OpenNebula/docs/pull/1232)
